src/Notepads/Package.appxmanifest

```diff
@@ -12,14 +12,14 @@
   IgnorableNamespaces="mp uap uap5 uap10 desktop4 iot2 rescap">
 
   <Identity
-    Name="Notepads"
+    Name="Notepads-Dev"
     Publisher="CN=jackil"
     Version="1.3.9.0" />
 
   <mp:PhoneIdentity PhoneProductId="dde28e6c-0f89-4885-83bc-51d9580080a1" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
 
   <Properties>
-    <DisplayName>Notepads App</DisplayName>
```

Comment:
I don't think you have to change this line right? Btw, I am not using Azure DevOps build to deploy to the store. I am suing local build so that I can set AppCenter secret before building the release. So in this case, I will have to make changes to switch it back from "Notepads - Dev" to "Notepads" right? Also I prefer "Notepads Dev" vs "Notepads - Dev" since it will be easier to search.

Reply:
> I don't think you have to change this line right? 

I have reverted it.
> Btw, I am not using Azure DevOps build to deploy to the store. I am suing local build so that I can set AppCenter secret before building the release. So in this case, I will have to make changes to switch it back from "Notepads - Dev" to "Notepads" right?

Also, you have to change the app execution alias from "Notepads-Dev" to "Notepads".
> Also I prefer "Notepads Dev" vs "Notepads - Dev" since it will be easier to search.

Made this change.